From e0b6c7e0bc5e1c36d7648091aef80d534fb5e50a Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 26 Sep 2017 15:58:45 -0700
Subject: [PATCH] joyent/node-triton#232 commit 8438f44 broke parseVolumeSize
 unit tests Reviewed by: Julien Gilli <julien.gilli@joyent.com> Approved by:
 Julien Gilli <julien.gilli@joyent.com>

---
 test/unit/parseVolumeSize.test.js | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/test/unit/parseVolumeSize.test.js b/test/unit/parseVolumeSize.test.js
index 024aa81..ab4eb4e 100644
--- a/test/unit/parseVolumeSize.test.js
+++ b/test/unit/parseVolumeSize.test.js
@@ -63,9 +63,6 @@ test('parseVolumeSize', function (tt) {
 
     tt.test('parsing valid sizes', function (t) {
         var validVolumeSizes = [
-            {input: '42m', expectedOutput: 42},
-            {input: '42M', expectedOutput: 42},
-            {input: '42g', expectedOutput: 42 * 1024},
             {input: '42G', expectedOutput: 42 * 1024}
         ];
 
-- 
2.21.0

